metricbeat/metricbeat.yml

```diff
@@ -60,16 +60,16 @@ setup.template.settings:
 # This requires a Kibana endpoint configuration.
 setup.kibana:
 
-# Kibana Host
-# Scheme and port can be left out and will be set to the default (http and 5601)
-# In case you specify and additional path, the scheme is required: http://localhost:5601/path
-# IPv6 addresses should always be defined as: https://[2001:db8::1]:5601
-#host: "localhost:5601"
+  # Kibana Host
```

Comment:
Is this related?

Reply:
no, it is a result of `make update` in other case tests were failing 🤷‍♀️